.gitignore

```diff
@@ -53,17 +53,17 @@ wheels/
 .installed.cfg
 *.egg
 src/*/version.info
-src/lightning_app/
-src/lightning_fabric/
-src/pytorch_lightning/
+src/lightning_app/*
+src/lightning_fabric/*
+src/pytorch_lightning/*
 !src/*/__about__.py
 !src/*/__main__.py
 !src/*/__setup__.py
 !src/*/__version__.py
 !src/*/MANIFEST.in
 !src/*/py.typed
 !src/*/README.md
-!src/*/*.info
+!src/*/shell-folder_code-lives-lightning.info
```

Comment:
What is this? I can't find any reference in our code for it.

Reply:
It is this strange file: https://github.com/Lightning-AI/lightning/blob/master/src/pytorch_lightning/shell-folder_code-lives-lightning.info